frontend/amundsen_application/static/js/pages/TableDetailPage/TableHeaderBullets/index.tsx

```diff
@@ -47,7 +47,12 @@ export class TableHeaderBullets extends React.Component<TableHeaderBulletsProps>
     const isViewCheck = isView === undefined ? false : isView;
     return (
       <ul className="header-bullets">
-        <li>{getDisplayNameByResource(ResourceType.table)}</li>
+        {database === 'feature' && (
+          <li>{getDisplayNameByResource(ResourceType.feature)}</li>
+        )}
+        {database !== 'feature' && (
+          <li>{getDisplayNameByResource(ResourceType.table)}</li>
+        )}
```

Comment:
A few questions:
1. what is `database` refers to here?
2. is `ResourceType.feature` means the Amundsen's feature, or our table feature?